test/appium/views/base_view.py

```diff
@@ -261,6 +261,18 @@ def click(self):
         self.driver.info('Tap on %s' % self.name)
 
 
+class OpenInStatusFromExternalBrowserButton(BaseButton):
+    def __init__(self, driver):
+        super(OpenInStatusFromExternalBrowserButton, self).__init__(driver)
+        self.locator = self.Locator.xpath_selector('//*[@text="Open in Status"] | //*[@content-desc="Open in Status"]')
+
+    def click(self):
+        for _ in range(5):
+            self.driver.swipe(500, 1000, 500, 500)
```

Comment:
we should swipe only in the test where the swipe is required

Reply:
Updated to swipe only once.